### DISCUSSION TITLE
Installed Homebrew - Getting Started

### DISCUSSION THREAD

**Discussion by Brewmeariver** [Homebrew/discussions#5021] [Getting started]
### Output of `brew config`

```shell
from where?
```

### Output of `brew doctor`

```shell
what?
```

### Description of issue

Hey I installed Xcode and CLT. After that I installed Homebrew with the installer from here (https://brew.sh/de/). 

This is the text from the installer. But if I tried to put the commands in the terminal it didnt work. Can anyone help me with it and give assistance?

BEFORE INSTALLATION

This package will install to:
	⁃	/opt/homebrew on Apple Silicon
	⁃	/usr/local/bin/brew and /usr/local/Homebrew on Intel

AFTER INSTALLATION 

Homebrew has enabled anonymous aggregate formulae and cask analytics.

Read the analytics documentation (and how to opt-out) here:
- https://docs.brew.sh/Analytics
No analytics data has been sent yet.

Homebrew is run entirely by unpaid volunteers. Please consider donating:
- https://github.com/Homebrew/brew#donations

Next steps:
- To add Homebrew to your PATH run brew shellenv in your shell profile (e.g. ~/.bash_profile or ~/.zprofile)
- Run brew help to get started
- Further documentation:
    - https://docs.brew.sh

**Comment by Brewmeariver** [comment]
I want to have an ICON in the FINDER and install SD which I already downloaded.

**Reply by SMillerDev** [reply]
Homebrew will not put an icon in finder since it is only a terminal application. I'm not sure what SD is.
